mirascope/cli/commands.py

```diff
@@ -23,7 +23,6 @@
         $ mirascope use my_prompt 0001
```

Comment:
it's starting to feel like time we refactor commands into a module with a separate file for each command. Future PR?

Reply:
Seems doable with a little additional thought. Will backlog this.